test/offline/manifest_convert_unit.js

```diff
@@ -265,6 +266,7 @@ describe('ManifestConverter', () => {
       mimeType: 'video/mp4',
       codecs: 'avc1.42c01e',
       frameRate: 22,
+      pixelAspectRatio: '16:9',
```

Comment:
Since this is a common display aspect ratio, I think it might mislead people into confusing DAR & PAR.  I was initially confused when I saw this, and had to do some reading to get it straight.

How about changing all of your 16:9 examples in the tests to something else?  A common non-square PAR seems to be 59:54.

Reply:
I changed it!